test/ANNOTATIONS.yaml

```diff
@@ -207,6 +207,8 @@ all:
     - Stack allocate on and task function argument bundles (#4074)
   12/08/16:
     - Re-enable denormalization (#4547)
+  12/12/16:
```

Comment:
Oh, nevermind about the 13th, there's a different commit for those (I just verified that the coforall one went in on the 13th)